pyspecdata/plot_funcs/pcolormesh.py

```diff
@@ -84,6 +84,7 @@ def pcolormesh(
         if ax1 is None:
             fig = plt.gcf()
             ax1 = fig.add_subplot(1, 1, 1)
+            one_plot = True
```

Comment:
The conditional here is incorrect. You also need to make sure that you are not doing real and imaginary data.

Reply:
Also, if there is just one plot, then you should set scale independently. There is no need for a different variable.

Reply:
if scale_independently is true then it won't allow setting vmin and vmax for intensity (see lines 131)

Reply:
also the conditional (if ax1 is None) is in master I can change it but this then won't be minimal changes. On line 74 there is an if for if the data is complex this section is when the data is not complex (which is what I want)